scriptmodules/admin/setup.sh

```diff
@@ -262,7 +262,15 @@ function package_setup() {
                 ;;
             X)
                 local text="Are you sure you want to remove $md_id?"
-                [[ "${__mod_section[$idx]}" == "core" ]] && text+="\n\nWARNING - core packages are needed for RetroPie to function!"
+                case "${__mod_section[$idx]}" in
+                    core)
+                        text+="\n\nWARNING - core packages are needed for RetroPie to function!"
+                        ;;
+                    depends)
+                        text+="\n\nWARNING - this package is required by other retropie packages - removing may cause other packages to fail."
```

Comment:
maybe capitalize `retropie` (RetroPie).

Reply:
Yeah. Sloppy text again from me.

Apart from this are you ok with it as I'll merge if so.

Reply:
yes, it looks fine.